src/Doxyfile.in

```diff
@@ -706,7 +700,7 @@ LAYOUT_FILE            =
 
 CITE_BIB_FILES         = @CMAKE_SOURCE_DIR@/biblio/bibliography.bib \
                          @CMAKE_SOURCE_DIR@/biblio/how_to_cite_cgal.bib \
-                         @CMAKE_SOURCE_DIR@/biblio/how_to_cite_gudhi.bib
+                         @CMAKE_CURRENT_BINARY_DIR@/biblio/how_to_cite_gudhi.bib
```

Comment:
This is not explained in the documentation (our fault), but we have to deal with 2 "versions" for gudhi:
* the dev version (the github repository)
* the "user" version (the .tar.gz you can download from the release)

We want to be able to build the documentation at least from the "user" version (maybe complicated from both versions).
To build the "user" version:
```bash
rm -rf build
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release -DWITH_GUDHI_PYTHON=OFF -DUSER_VERSION_DIR=my_version ..
make user_version
cd my_version
```
From here, you will see that the biblio file is in the `@CMAKE_SOURCE_DIR@` (and not in `@CMAKE_CURRENT_BINARY_DIR@`).
From an external point of view, it is normal you felt into this trap and sorry about that, but I would keep it as it is.

Reply:
I downloaded the 3.5.0 version of GUDHI and I see that the file `CMakeGUDHIVersion.txt` is here and also the file `biblio/how_to_cite_gudhi.bib` but not the file `biblio/how_to_cite_gudhi.bib.in`.
What is the objection against distributing the `biblio/how_to_cite_gudhi.bib.in` besides the file `biblio/how_to_cite_gudhi.bib` as well so that during the build procedure the file `build/biblio/how_to_cite_gudhi.bib` is generated and can be used for e.g. the documetation generation?

Reply:
That's a good point. Maybe your solution is better. To be tested (maybe not on this PR)

Reply:
Thanks.
As I don't have a full version (e.g. Boost) and I don't know how the distribution is created it will not be possible for me to test it so I leave this point to you / the GUDHI project.

Reply:
Thanks, I tracked your point on #612